inspirehep/utils/record.py

```diff
@@ -31,3 +35,53 @@ def get_title(record):
             return title['title']
 
     return title
+
+
+def get_smart_value(record, key, default=None):
```

Comment:
Hmm, what about `get_dotted_value`? Otherwise `get_value` could work as well. The smartness is only performed upon asking for it.